test/errorHandler/ErrorHandlerSpec.scala

```diff
@@ -54,7 +54,7 @@ class ErrorHandlerSpec extends UnitSpec with GuiceOneAppPerSuite with Injecting{
       val docHeading = doc.select("#errorHeading").text()
       val docMessage = doc.select("#errorText").text()
 
-      docTitle shouldBe title + " GOV.UK"
+      docTitle shouldBe s"$title - Marriage Allowance - GOV.UK GOV.UK"
```

Comment:
This is wrong - GOV.UK should not be there twice. If this test is passing, you may have doubled the GOV.UK part in your page titles.

Reply:
Fixed